libafl/src/executors/forkserver.rs

```diff
@@ -1157,7 +1170,12 @@ where
 
         if let Some(status) = self.forkserver.read_st_timed(&self.timeout)? {
             self.forkserver.set_status(status);
-            if libc::WIFSIGNALED(self.forkserver().status()) {
+            let exitcode_is_crash = if let Some(crash_exitcode) = self.crash_exitcode {
+                libc::WSTOPSIG(self.forkserver().status()) == crash_exitcode
```

Comment:
wfisignaled doesn't work here?

Reply:
``WIFSIGNALED`` returns a bool so we can't  compare it to a given exit code

Reply:
https://github.com/AFLplusplus/AFLplusplus/blob/1d17210d9fb0eb37ba866a3697643a9e4f37acd5/src/afl-forkserver.c#L2045

AFL++ uses `WEXITSTATUS` (and it's `i8`/`u8`, not `i32`?)

Reply:
Should we keep this closer to AFL++? @vanhauser-thc for visibility

Reply:
@domenukk yeah you're right. its i8.

Reply:
i think it should be close to AFLpp